libs/application-generic/src/usecases/execute-bridge-request/execute-bridge-request.usecase.ts

```diff
@@ -272,4 +241,135 @@ export class ExecuteBridgeRequest {
 
     return apiUrl;
   }
+
+  private handleResponseError(error: unknown, url: string) {
+    if (error instanceof RequestError) {
+      let body: Record<string, unknown>;
+      try {
+        body = JSON.parse(error.response.body as string);
+      } catch (e) {
+        // If the body is not valid JSON, we'll just use an empty object.
+        body = {};
+      }
+
+      if (
+        error instanceof HTTPError &&
+        Object.values(ErrorCodeEnum).includes(body.code as ErrorCodeEnum)
+      ) {
+        // Handle known Bridge errors. Propagate the error code and message.
+        throw new HttpException(body, error.response.statusCode);
+      } else if (body.code === TUNNEL_ERROR_CODE) {
```

Comment:
Readability tip, since you are throwing, you don't need the `else if`. You can put an if in a new line.

Reply:
Great point! I will update.